### ISSUE TITLE
superset fab list-users return username, email, role but not firstname, lastname, active

### ISSUE THREAD

**Issue by xavier-GitHub76** [apache/superset#28691]
### Bug description

Hello, 

superset fab list-users list all users on the database
but all colums are not returned.

username, email, role are listed but not firstname, lastname, active

colums in database are : 
![image](https://github.com/apache/superset/assets/82046143/cc5d43b0-9ef3-4283-9494-ae579afe37ab)

Best regards

### How to reproduce the bug

superset fab list-users

### Superset version

master / latest-dev

### Python version

3.9

### Node version

16

### Browser

Chrome

### Checklist

- [ ] I have searched Superset docs and Slack and didn't find a solution to my problem.
- [ ] I have searched the GitHub issue tracker and didn't find a similar bug report.
- [ ] I have checked Superset's logs for errors and if I found a relevant Python stacktrace, I included it here as text in the "additional context" section.

**Comment by xavier-GitHub76** [comment]
@dosu where the superset fab list-users command is defined ?

**Comment by xavier-GitHub76** [comment]
@dosu I find superset/cli/main.py file but not superset/cli/fab.py file

**Comment by rusackas** [comment]
@dpgaspar might know if this is a bug or intentional by design for some reason.

**Comment by rusackas** [comment]
Is this still the case in 4.1.1? I seem to recall a PR that added some of these fields, but I haven't gone to check yet.

**Comment by xavier-GitHub76** [comment]
It's present on 5.0.0RC1

![Image](https://github.com/user-attachments/assets/525237f3-1271-4068-b192-140a3d367dc9)

**Comment by rusackas** [comment]
Fair enough! Thanks for the update. I'm tempted to move this to an "Ideas" thread since it's more feature request than bug, but I'll leave it here for now. PRs welcomed from anyone on the thread, as this isn't likely to be prioritized by the usual core committers.

**Comment by xavier-GitHub76** [comment]
I create a pull request on Flask-AppBuilder https://github.com/dpgaspar/Flask-AppBuilder/pull/2369
